src/agentlab/agents/generic_agent/generic_agent.py

```diff
@@ -99,8 +97,8 @@ def get_action(self, obs):
             # cause it to be too long
 
             chat_messages = [
-                SystemMessage(content=system_prompt),
-                HumanMessage(content=prompt),
+                {"role": "system", "content": system_prompt},
```

Comment:
perhaps we can make some utils to help user make prompts, but maybe that would
be "holding their hands" too much. 

e.g.
```python
chat_messags = [
    make_system_prompt(system_prompt)
    make_user_prompt(prompt)
]
```

Just brainstorming, for now we can leave it like this